spec/models/user_spect.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
- [ ] Kindly add model specs, this means that we should test the creation of a user with valid and invalid attributes.
- [ ] Let's rename our files to end with `spec` instead of `spect`

Reply:
Hi @elisha2kyakpo1 you right i will change spect to spec, but about comment and link before create  any method   how can i  create spec method for them please check the requirement

Reply:
@samiullahbahadur, it's a requirement which you have also highlighted in the last part of your PR description.
`Add unit specs for all of your models' methods and validations.`

Reply:
Hi @elisha2kyakpo1  you right , I have fix it that